INSTRUMENTS/LABJACK/LABJACKU3/LABJACKU3.py

```diff
@@ -1,60 +1,41 @@
-from flojoy import flojoy, DataContainer
+from flojoy import flojoy, OrderedPair, DataContainer
 import LabJackPython
 import u3  # Import the library from LabJackPython in order to use our U3-LV device
 
 
-@flojoy
-def LABJACKU3(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
-    """Takes a number of sensors as parameters and return their temperature measurement"""
+@flojoy(deps={"labjackpython": "2.1.0"})
+def LABJACKU3(default: OrderedPair = None, numbers: int = 1) -> OrderedPair:
+    """
+    The node LABJACKU3 allows you to record and return temperature measurements with a LABJACK U3 device.
+
+    Parameters
+    -----------
+    number: int
+         Defines the number of temperature sensors connected to the LabJack U3 device.
+    """
 
     voltages: list[float] = []
     temperatures: list[float] = []
     temperatures_celsius: list[float] = []
     sensor_num: list[int] = []
-    sensor_number: int = params["numbers"]
+    sensor_number: int = numbers
```

Comment:
Instead of assigning numbers to `sensor_number`, can we rename the parameter name `numbers` to `sensor_number` ?